source/features/forked-to.tsx

```diff
@@ -61,6 +61,7 @@ async function updateUI(forks: string[]): Promise<void> {
 				<ChevronRightIcon className="v-align-text-top"/>
 			</a>,
 		);
+		forkButton.style.cssText = 'margin-right:0px !important';
```

Comment:
I used `style.cssText` to be able to set the styling as important, to override the other CSS rules. Setting `cssText` will overwrite the current element's inline styling - but GitHub uses stylesheets, meaning that there's no CSS that will be overwritten.